src/openloco/windowmgr.cpp

```diff
@@ -789,6 +911,96 @@ namespace openloco::ui::windowmgr
         }
     }
 
+    static loco_global<int16_t, 0x01136268> _1136268;
+    static loco_global<uint16_t[1], 0x0113626A> _113626A;
```

Comment:
Why are these arrays?

Reply:
To be fair, I have no idea as to what this function even aims to do.